<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxyWindow</class>
 <widget class="QWidget" name="ProxyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>代理设置</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableWidget" name="tableWidget_proxy"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
